### PR TITLE
Fix AttributeError when CONTENT_ORIGIN is undefined

### PR DESCRIPTION
**Pull request by daviddavis**
fixes #8235

### CODE REVIEW

**Comment by daviddavis** [comment]
@bmbouter does this fix makes sense or is there a better way to solve this?

File: pulp_ansible/app/settings.py [hunk 1]
```diff
@@ -21,4 +21,4 @@
 }
 
 ANSIBLE_API_HOSTNAME = "http://" + socket.getfqdn()
-ANSIBLE_CONTENT_HOSTNAME = settings.CONTENT_ORIGIN + "/pulp/content"
+ANSIBLE_CONTENT_HOSTNAME = getattr(settings, "CONTENT_ORIGIN", "") + "/pulp/content"
```
Comment (fao89) [inline pulp_ansible/app/settings.py:24]:
```suggestion
ANSIBLE_CONTENT_HOSTNAME = getattr(settings, "CONTENT_ORIGIN", "http://" + socket.getfqdn()) + "/pulp/content"
```

Comment (bmbouter) [reply pulp_ansible/app/settings.py:24]: I don't think we can do this because we can't assume the http protocol.

**Comment by daviddavis** [comment]
Closing in favor of https://github.com/pulp/pulpcore/pull/1119.

**Pull request closed without merging**